From 0db95bf34a1a4b87a76d1979c80896bf6c865ab0 Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Tue, 28 Oct 2014 18:45:19 +0100
Subject: [PATCH 075/192] profile-resetter: disable send-settings

When the user wishes to reset his preferences, Chromium would suggest
to send over the old settings to Google. I presume that this is for
Google to investigate into the setting combinations that led the user
to issue a rest.

Of course, we cannot let that happen by default. Google is not to get
any information.

License: GPL-3.0-only - https://spdx.org/licenses/GPL-3.0-only.html
---
 chrome/browser/profile_resetter/reset_report_uploader.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/browser/profile_resetter/reset_report_uploader.cc b/chrome/browser/profile_resetter/reset_report_uploader.cc
index aae06fa117a36..0272c59ab0480 100644
--- a/chrome/browser/profile_resetter/reset_report_uploader.cc
+++ b/chrome/browser/profile_resetter/reset_report_uploader.cc
@@ -20,7 +20,7 @@
 
 namespace {
 const char kResetReportUrl[] =
-    "https://sb-ssl.google.com/safebrowsing/clientreport/chrome-reset";
+    "about:blank"; //https://sb-ssl.google.com/safebrowsing/clientreport/chrome-reset";
 
 GURL GetClientReportUrl(const std::string& report_url) {
   GURL url(report_url);
-- 
2.25.1

